docs/FHSS_SHAPING.md

```diff
@@ -0,0 +1,43 @@
+# mLRS Documentation: FHSS Shaping #
+
+([back to main page](../README.md))
+
+mLRS uses frequency hopping. 
+
+The number of frequencies which are used in the hopping depend on the RF band and the selected mode. For instance, in the 2.4 GHz band and for the 50 Hz mode, the FHSS sequence consists of 24 different frequencies. The frequency values and their sequence however are determined by the bind phrase. This allows for one to operate several mLRS systems in the same RF band with minimized mutual interference, by choosing different bind phrases for each system.
```

Comment:
Perhaps:
- 'The number of frequencies used in the FHSS sequence...'
- 'The frequency values and sequence are determined by the bind phrase.'
- Reverse, e.g. 'Choosing different bind phrases for each mLRS system allows one to operate several systems in the same RF band with minimized mutual interference.'

Reply:
THX! done.